docker/content.md

```diff
@@ -18,15 +18,13 @@ If you are still convinced that you need Docker-in-Docker and not just access to
 
 ## Start a daemon instance
 
-**IMPORTANT:** this image defaults to `--storage-driver=vfs`, which will be very slow and inefficient (but is the only driver which is guaranteed to work regardless of your underlying filesystem). Which driver you should use varies depending on your needs, but a good rule of thumb is that your DinD instance should be using the same driver as your host (which can be seen under `Storage Driver` in the output of `docker info`). See the "Custom daemon flags" section below for how to specify your storage driver.
```

Comment:
FTR, https://github.com/docker-library/docker/pull/89 is the PR which removed this. :+1: :innocent: